lib/travis/build/script/r.rb

```diff
@@ -601,7 +601,8 @@ def normalized_r_version(v=Array(config[:r]).first.to_s)
           when 'bioc-devel'
             config[:bioc_required] = true
             config[:bioc_use_devel] = true
-            normalized_r_version('devel')
```

Comment:
Is there a reason you aren't using `normalized_r_version('release')`?

Reply:
I was just using the code in `when 'bioc-release'` as an example. I can change lines 605 & 610 to `normalized_r_version('release')`

Reply:
it could also be `r_latest`